phpstan.neon

```diff
@@ -1,5 +1,5 @@
 parameters:
-	level: max
+	level: 5
```

Comment:
not sure about lowering the level here, will check later

Reply:
Another option would be to have level max with a large baseline for now.

Reply:
I pushed a second commit with the phpstan level of max  and a baseline that could then be resolved step by step